code/game/objects/items/kitchen.dm

```diff
@@ -96,6 +96,23 @@
 	wound_bonus = -5
 	bare_wound_bonus = 10
 	tool_behaviour = TOOL_KNIFE
+	var/armor_pene_stab = 20
+	var/force_stabby = 5
+	var/wound_bonus_stabby = 5
```

Comment:
Why only have this be on kitchen knives? IMO this should be made generic to any sharp object. Maybe go a step further and add a way to track objects that are both sharp and pointy.

Reply:
Yeah now that I think about it I found out that many items that are also small and sharp arent used, though theres also a problem that I am a newbie so yeah 😅

Reply:
> generic to any sharp object

It would not make sense to have this for axes and things like that

Reply:
> It would not make sense to have this for axes and things like that

I agree, hence: 
> Maybe go a step further and add a way to track objects that are both sharp and pointy.

Reply:
making knifes more different